app/assets/javascripts/app.js

```diff
@@ -0,0 +1,49 @@
+
+$(function() {
+    $("form").on("submit", function(e) {
+       e.preventDefault();
+       // prepare the request
+       var request = gapi.client.youtube.search.list({
+            part: "snippet",
+            type: "video",
+            q: encodeURIComponent($("#search").val()).replace(/%20/g, "+"),
+            order: "viewCount"
+       }); 
+       // execute the request
+       request.execute(function(response) {
+          var results = response.result;
+          $("#results").html("");
+          $.each(results.items, function(index, item) {
+            $("#results").append(" <h3 class='video_title yt'> "+item.snippet.title+"</h3>");
+            $("#results").append(" <iframe class='vide_container yt' width='400' height='200' frameborder='0' src='http://www.youtube.com/embed/"+item.id.videoId+"' allowfullscreen></iframe> ")
+          });
+          resetVideoHeight();
+       });
+    });
+    $(window).on("resize", resetVideoHeight);
+});
+
+ // Responsive Desing Videos
+function resetVideoHeight(){
+  $(".video").css("height", $("#results").width() * 9/16);
+}
+
+function init(){
+  gapi.client.setApiKey("AIzaSyD3ujw52WSOKZ0jiIYZgcoudXD534yQlcQ");
+  gapi.client.load("youtube", "v3", function(){
+    // youtube API loaded
+  });
+}
+
+function change_content(num){
```

Comment:
This function does the same in both conditionals, Why doesn't you use one condition? Something like that:
If (num === 1 || num == 2) { ... some code }